docs/interactive-scroll-view.md

```diff
@@ -0,0 +1,40 @@
+---
+id: interactive-scroll-view
+title: ScrollView 
+sidebar_label: ScrollView
+---
+
+Render `children` elements in a scrollable container with fixed height, using mouse scroll events. Useful for reporters and logs display.
+
+## Props
+
+* `children: React.Element<*>` - Content to be rendered and scrolled.
+* `height: number` - Fixed height of the visible area.
```

Comment:
Is it possible to use the container height instead of requiring to pass a `height` prop?

Also isn't it better to use the `height` from `style`? It'll be confusing when you've a `height` prop and `height` style.

Reply:
That's actually a good point. It could be possible I guess. The initial implementation was without `style` but I thought that it would be good to pass it as well thus the inconsistency. It might be a good idea to fall back to container height if the height is not defined. Not sure. But good point.

Reply:
If the parent has a height specified already/or it's the top level element so it's the terminal height, it'll be better to use that one.

Reply:
Using parent's height if specified (so like few % of cases) will needlessly increase complexity. Not gonna do it right now, but after integrating Reflow.